bin/env_variables_all.txt

```diff
@@ -144,3 +144,6 @@ XUI_LAUNCH_DARKLY_CLIENT_ID=1
 DB_USE_SSL=false
 ES_ENABLED_DOCKER=false
 IDAM_KEY_CCD_PS=AAAAAAAAAAAAAAAA
+IDAM_KEY_NEXT_HEARING_UPDATER=AAAAAAAAAAAAAAAA
+CCD_NEXT_HEARING_DATE_PASSWORD=Pa55word11
```

Comment:
CCD_NEXT_HEARING_DATE_UPDATER_USER_PWD ??

Reply:
I could change it but this would lead to change the original HMAN-320/326. This is user/role defined already.